web/src/beta/lib/core/engines/Cesium/Feature/Marker/index.tsx

```diff
@@ -25,15 +25,25 @@ import {
 
 import marker from "./marker.svg";
 
-export type Props = FeatureProps<Property>;
+export type Props = FeatureProps<Property> & {
+  hideIndicator?: boolean;
```

Comment:
You can get this value from `property`: https://github.com/reearth/reearth/pull/866/files#diff-48349bf003319b11ce3f0c81066fa84eb0a7978cc431067d4b9009b267cd0064R92
So you don't need to define it in here.